contracts/YearnHarvest.sol

```diff
@@ -175,6 +177,22 @@ contract YearnHarvest {
         uint256 callCostInWei = 1e8; // low value so that the trigger focuses on all other conditions
 
         // Check active strategies and return the first one that is ready to harvest.
+
+        // REVIEW: Two issues of iterating over ALL strategies I can think of:
+        // 1) You might run out of gas. If there are enough reverting harvest
+        // strategies, the method might consume all the gas.
+        //
+        // 2) You are forcing an order in the harvest procedure.
+        // Not an issue per se, but might be an attack vector
+        //
+        // I would prefer this logic to live on keepers logic, perhaps offering
+        // this helper method but separated in two:
+        // a) getAllHarvestableStrategies()
+        // b) CheckIfHarvestable()
+        //
+        // On the other hand, we know that keepers check in a fork before sending
```

Comment:
Let's discuss. I see we have a gap in understanding how Gelato Ops work. If it's not clear in the article, I have work to do.